config/configtls/README.md

```diff
@@ -72,6 +72,14 @@ Additionally certificates may be reloaded by setting the below configuration.
    Accepts a [duration string](https://pkg.go.dev/time#ParseDuration),
    valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 
+- `trusted_keys` (optional) : TrustedKeys allows specifying a list of SHA256 hashes to trust when verifying client certificates. This allows only trusting a specific set of keys, rather than all the keys signed by a given CA. Note: The client certificate still has to pass standard TLS verification, so you must still set the required CA values.
```

Comment:
We don't necessarily need to implement this now, but if we want to change the hashing algorithm used here, how do you see that transition happening? I guess we could have something like this:
```yaml
tls:
  trusted_keys:
    hashing_function: sha256
    hashes:
      - "29:9D:39:8E:26:E7:72:88:D7:5D:2D:34:FE:8C:41:FA:9F:F1:C3:D7"
```

or instead each element of `tls::trusted_keys` could be something like this:

```yaml
tls:
  trusted_keys:
    - sha256: "29:9D:39:8E:26:E7:72:88:D7:5D:2D:34:FE:8C:41:FA:9F:F1:C3:D7"
```

What do you think is the best approach? I am leaning towards the latter option (it's a bit more flexible)